modules/aws/config.go

```diff
@@ -0,0 +1,82 @@
+package aws
+
+//Reference for mocking purposes https://github.com/aws/aws-sdk-go/blob/master/aws/config.go
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+)
+
+type RequestRetryer interface{}
+
+type Config struct {
```

Comment:
This needs a clearer name, like `AwsConfigOverrides`. Also, is there not a `structure` in the AWS SDK that already has all of these defined?

Reply:
Yeah it is in the SDK but you know trying to avoid the new depency inside the test I think could be a good Idea